### DISCUSSION TITLE
Activate Page 0

### DISCUSSION THREAD

**Discussion by eboston** [HASwitchPlate/HASPone#22] [Q&A] [answered]
The Activate Page blueprint doesn't allow you to switch to page 0 with the WiFi connection info.  Is there a way to do this?

**Comment by aderusha** [comment, marked as answer]
There is now!  Didn’t occur to me at the time that people would want to target page 0, but that is a reasonable thing to do, and I just changed the blueprint to accommodate your request.

**Reply by eboston** [reply]
That got it.  Thanks.
